docs/user/tutorials/01-10-mtls-authentication/configure-mtls-Gardener-certs.md

```diff
@@ -0,0 +1,547 @@
+      namespace: test
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: httpbin
+      namespace: test
+      labels:
+        app: httpbin
+        service: httpbin
+    spec:
+      ports:
+      - name: http
+        port: 8000
+        targetPort: 80
+      selector:
+        app: httpbin
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: httpbin
+      namespace: test
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: httpbin
+          version: v1
+      template:
+        metadata:
+          labels:
+            app: httpbin
+            version: v1
+        spec:
+          serviceAccountName: httpbin
+          containers:
+          - image: docker.io/kennethreitz/httpbin
+            imagePullPolicy: IfNotPresent
+            name: httpbin
+            ports:
+            - containerPort: 80
+    EOF
+    ```
+
+12. To expose the sample HTTPBin Deployment, create an APIRule custom resource. 
+    The APIRule must include the `X-CLIENT-SSL-CN: '%DOWNSTREAM_PEER_SUBJECT%'`, `X-CLIENT-SSL-ISSUER: '%DOWNSTREAM_PEER_ISSUER%'`, and `X-CLIENT-SSL-SAN: '%DOWNSTREAM_PEER_URI_SAN%'` headings. These headers are necessary to ensure that the backend service receives the authenticated client's identity.
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: gateway.kyma-project.io/v2
+    kind: APIRule
+    metadata:
+      name: httpbin-mtls
+      namespace: test
+    spec:
+      gateway: test/kyma-mtls-gateway
+      hosts:
+        - "${WORKLOAD_DOMAIN}"
+      rules:
+        - methods:
+            - GET
+          noAuth: true
+          path: /*
+          timeout: 300
+          request:
+            headers:
+              X-CLIENT-SSL-CN: '%DOWNSTREAM_PEER_SUBJECT%'
+              X-CLIENT-SSL-ISSUER: '%DOWNSTREAM_PEER_ISSUER%'
+              X-CLIENT-SSL-SAN: '%DOWNSTREAM_PEER_URI_SAN%'
+      service:
+        name: httpbin
+        port: 8000
+    EOF
+    ```
+
+13. Test the mTLS connection.
+    
+    1. Run the following curl command:
+    
+    ```bash
+    curl --fail --verbose \
+      --key "${CLIENT_CERT_KEY_FILE}" \
+      --cert "${CLIENT_CERT_CRT_FILE}" \
+      "https://${WORKLOAD_DOMAIN}/headers?show_env==true"
+    ```
+
+    If successful, you get code `200` in response. The **X-Forwarded-Client-Cert** heading contains your client certificate.
+    
+    2. To thest the connection using your browser, import the client certificates into your operating system or browser. For Chrome, you can use the generated PKCS#12 file. Then, open `https://{WORKLOAD_DOMAIN}`.
+
+### **Default Domain**
+
+
+1. Create a namespace with enabled Istio sidecar proxy injection.
+   
+    ```bash
+    kubectl create ns test
+    kubectl label namespace test istio-injection=enabled --overwrite
+    ```
+2. Export the following domain names as enviroment variables. Replace `my-own-domain.kyma.ondemand.com` with the name of your domain.
```

Comment:
I'm more in favour of actually giving the user a kubectl command to get the default domain. Otherwise the `my-own-domain` part is confusing with custom domain also in scope of this tutorial.